driver/csiplugin/controllerserver.go

```diff
@@ -385,7 +391,7 @@ func (cs *ScaleControllerServer) createFilesetBasedVol(ctx context.Context, scVo
 		}
 		scVol.ParentFileset = ""
 		createDataDir := false
-		_, err = cs.createFilesetVol(ctx, scVol, indepFilesetName, fsDetails, opt, createDataDir, true, isNewVolumeType)
+		_, err = cs.createFilesetVol(ctx, scVol, indepFilesetName, fsDetails, opt, createDataDir, true, isCGVolume, bucketInfo)
```

Comment:
for CGVolume, bucketInfo shouldn't be passed?

Reply:
Yes, it's needed and processed only for cache volume, made this change.